modin/backends/pandas/query_compiler.py

```diff
@@ -2958,8 +2958,8 @@ def sort_rows_by_column_values(self, columns, ascending=True, **kwargs):
         PandasQueryCompiler
             A new query compiler that contains result of the sort
         """
-        na_position = kwargs.get("na_position", "last")
-        kind = kwargs.get("kind", "quicksort")
+        ignore_index = kwargs.get("ignore_index", False)
```

Comment:
Could we handle `ignore_index` at the API layer? Or do you think it is better here?

Reply:
Makes sense.